src/test/java/data/TaskManagerTest.java

```diff
@@ -128,7 +137,7 @@ void addDeadline_validInput_addsTask() throws TaskManagerException {
         // Arrange
         LocalDate date = LocalDate.now();
         String taskDescription = "Test Deadline";
-        String byDate = "05/04/2024";
+        String byDate = LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yyyy"));
```

Comment:
Good idea to update the tests to use the current date whenever the test is run instead of hard coded dates!